src/main/java/com/redhat/rcm/offliner/Main.java

```diff
@@ -183,6 +182,9 @@ else if ( result.isAvoided() )
                 }
             }
 
+            Set<String> pomPaths = new HashSet<>();
+            File download = new File( opts.getDownloads().getAbsolutePath() );
```

Comment:
I think this is already a File, so do we need to construct a new one? Or, if you need an absolute one, can you just use getAbsoluteFile()?